src/mod/dns.mod/coredns.c

```diff
@@ -38,6 +38,8 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <arpa/nameser.h>
+#undef answer /* before resolv.h because it could collide with src/mod/module.h
```

Comment:
shouldn't this be surrounded by ifdef then?

Reply:
why? doesn't undef work in any case? then i would of course have to fix a bug in my patch.